gui-docker

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash -u
+
```

Comment:
I think we need to specify the source of the script
https://github.com/ros-planning/moveit2/blob/main/.docker/gui-docker

Reply:
Or does it makes sense to send a PR to the moveit2 repo directly?

Reply:
I am not sure. The main different is how to check if there is nvidia card.

moveit2 use `lspci | grep -i "vga .*nvidia"`. But it doesn't work for me.

So I changed to `lspci | grep -i "nvidia" `